### ISSUE TITLE
Forum example on website should not use bigint for exp type

### ISSUE THREAD

**Issue by markhalonen** [graphile/postgraphile#1166]
I'm submitting a

* [ x ] bug report
* [ ] feature request
* [ ] question 

Here, the type of a jwt `exp` is `bigint`: 

https://www.graphile.org/postgraphile/postgresql-schema-design/#logging-in

I think https://github.com/graphile/graphile-engine/pull/376 caused `bigint`'s to be strings, so when lodash's `isNumber` checks the string, I got the error `"exp" should be a number of seconds`. 

Changing the type to `int` instead of `bigint` for `exp` works, but also the max value of `int` in postgres is `2147483647`, which is in 2038.

**Comment by benjie** [comment]
Reproduced with:

```
$ createdb issue1166
$ psql issue1166
[issue1166] # create type jwt_token as (role text, person_id int, exp bigint);
CREATE TYPE
[issue1166] # create function auth() returns jwt_token as $$ select ('foo', 2147483647, 999999999999)::jwt_token;$$ language sql;
CREATE FUNCTION
[issue1166] # 
```

```graphql
mutation {
  auth(input:{}) {
    jwtToken
  }
}
```

```
node ./build/postgraphile/cli.js  --token public.jwt_token -c issue1166 --jwt-secret secret --extended-errors severity,code,detail,hint,position,internalPosition,internalQuery,where,schema,table,column,dataType,constraint,file,line,routine
```

```json
{
  "errors": [
    {
      "extensions": {
        "exception": {}
      },
      "message": "\"exp\" should be a number of seconds",
      "locations": [
        {
          "line": 3,
          "column": 5
        }
      ],
      "path": [
        "auth",
        "jwtToken"
      ]
    }
  ],
  "data": {
    "auth": {
      "jwtToken": null
    }
  }
}
```

**Comment by benjie** [comment]
Hey @markhalonen @pohy; are there any other related issues in JWTs - anywhere else where a bigint needs to be an int rather than a string? (I don't use JWTs.)

Fixed in https://github.com/graphile/graphile-engine/pull/542

**Comment by markhalonen** [comment]
Thanks for the quick response and fix! Unfortunately GitHub doesn't support Organizations sponsoring, but I've sponsored you on Patreon!

> anywhere else where a bigint needs to be an int rather than a string? (I don't use JWTs.)

I'm not aware of any other locations where this is an issue.

> Hey folks; I've spent much time working on a batteries-included MIT-licensed starter kit for PostGraphile.

I saw this on Discord, this is definitely something that will help people get started. Following the tutorial is a daunting task, but it does teach you a lot,  but a quickstart script would demonstrate value faster.

Anyways, thanks for maintaining this excellent software!
